Cargo.toml

```diff
@@ -26,3 +26,6 @@ members = [
 codegen-units = 1
 lto = "fat"
 panic = "abort"
+
+[patch.crates-io]
+cynic = { git = "https://github.com/xgreenx/cynic", branch = "feagure/expose-steaming-operation" }
```

Comment:
Looks like your [PR has been merged](https://github.com/obmarg/cynic/commit/aff0aed74cfd21c87dcbde63351a862285106e9c) and we can switch back to master awaiting release